x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/types.ts

```diff
@@ -26,4 +26,6 @@ export interface GraphEdge {
   isOrigin: boolean;
   isOriginAlert: boolean;
   isAlert: boolean;
+  actorsDocData?: Array<string | null> | string;
```

Comment:
_nit_: since the query checks for `actor.entity.id IS NOT NULL` the proper type here could be `Array<string> | string`.

as for `targetsDocData`, I'm not sure either if you'd get array of null when target is not available. worth to double check if it reduces type checking.